staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers.go

```diff
@@ -282,12 +272,34 @@ func (w *deferredResponseWriter) unbufferedWrite(p []byte) (n int, err error) {
 		w.w = hw
 	}
 
+	span := tracing.SpanFromContext(w.ctx)
+	span.AddEvent("About to start writing response",
+		attribute.String("writer", fmt.Sprintf("%T", w.w)),
```

Comment:
I'd still probably include the size of the initial write... in non-streaming cases, that's the entire object

`attribute.Int("size", len(p))`

in streaming cases, we'll see the difference between the initial write and the total at the end, which is also interesting

Reply:
Updated with "size"